general/1_terminal/Tania-Morozova/Task_4.txt

```diff
@@ -0,0 +1 @@
+cat test_json.json | jq '.messages[] .text'
```

Comment:
If you'd like an additional assignment, you may try to make the output prettier.

Currently, it renders the forwards as arrays.